Backend/credentials/models.py

```diff
@@ -24,12 +24,17 @@ class CandidateConfig(models.Model):
     resume = models.BinaryField(max_length=None, null=True)#If there are errors uploading documents to this column, might need to set editable=True as a parameter
     current_employer = models.ForeignKey(Employer, default=1, on_delete=models.CASCADE, null=True)
 
+    def __str__(self):
+        return self.candidate.__str__()
```

Comment:
Couldn't this just be return self.candidate since we defined the __str__ method in the candidate model?

Reply:
No because you still need to call the str method oft the other object because otherwise it simply returns the object. It's not because it's in the __str__method that it automatically converts it to a string.

Reply:
im pretty sure it's only the python print() that is smart enough to find __str__() automatically. else I would probably just get a memory address to the object if i do client.candidate